adding -fcommon flag since it was removed from defaults in newer compilers
author: Denis Kristak (INUITS)
diff -ruN PPanGGOLiN-1.1.136_orig/setup.py PPanGGOLiN-1.1.136/setup.py
--- PPanGGOLiN-1.1.136_orig/setup.py	2021-11-23 10:32:07.957348000 +0100
+++ PPanGGOLiN-1.1.136/setup.py	2021-11-23 10:43:58.316011000 +0100
@@ -28,7 +28,8 @@
                 "Programming Language :: Python :: 3",
                 "Topic :: Scientific/Engineering :: Bio-Informatics"],
         entry_points={"console_scripts":["ppanggolin = ppanggolin.main:main"]},
-        ext_modules = [Extension(name = "nem_stats",sources =[NEM_DIR_PATH+'nem_stats.pyx',
+        extra_compile_args=['-fcommon'],
+        ext_modules = [Extension(name = "nem_stats", extra_compile_args=['-fcommon'], sources =[NEM_DIR_PATH+'nem_stats.pyx',
                                                                         NEM_DIR_PATH+'nem_exe.c',
                                                                         NEM_DIR_PATH+'nem_alg.c',
                                                                         NEM_DIR_PATH+'nem_nei.c',
